.github/workflows/check-code.yml

```diff
@@ -1,6 +1,5 @@
 name: "Check code"
 on:
-  push:
```

Comment:
if we remove this the contributors will not be able to check errors before the PR

Reply:
Confirmed with @bodymindarts that we'd like all work in progress to be in PRs. If it's not ready, make it a draft PR.

Reply:
Draft PRs will also get feedback

Reply:
OK but draft PRs to main (it will not work for PR with other base ). I mention this because you are using that for the lastest changes